jobs/build/scan-images/Jenkinsfile

```diff
@@ -17,18 +17,26 @@ properties([[
 
 node('openshift-build-1') {
   try {
-    stage('clone') { dir('aos-cd-jobs') { checkout scm } }
+    stage('clone') {
+      dir('aos-cd-jobs') { checkout scm }
```

Comment:
What if these directories don't exist? Do we need to rm -rf them and mkdir them before we clone?

Reply:
I never thought about it, but it has worked on all my tests, with a repo that
I'm constantly updating.  I tested it again to be sure, the directory is
created if it doesn't exist and is reused if it does.  A `git fetch` is
executed to update the repository and, since we're not modifying any of them, I
don't think there will be a problem.